test/uk/gov/hmrc/cdsreimbursementclaimfrontend/controllers/claims/BankAccountControllerSpec.scala

```diff
@@ -324,7 +325,7 @@ class BankAccountControllerSpec
 
     "Personal Bank Account" should {
 
-      val personalBankAccount = sample(genPersonalBankAccountDetails)
+      val personalBankAccount = sample(arbitraryPersonalBankAccountDetails)
```

Comment:
we should reference the `Arbitrary[T]` here - the whole point is to let magnolia work out which arbitrary it needs to pick and generate a value for a type `T` - all you need to do here is `sample[BankAccountDetails]` and we are good :+1: